conda_forge_tick/make_graph.py

```diff
@@ -154,14 +156,18 @@ def make_graph(names, gx=None):
     env = builtins.__xonsh__.env
     debug = env.get("CONDA_FORGE_TICK_DEBUG", False)
     builder = _build_graph_sequential if debug else _build_graph_process_pool
-    builder(gx, total_names, new_names)
+
+    for i in range(0, ceil(len(total_names)/BATCH_SIZE)):
+        names = total_names[i*BATCH_SIZE:(i+1)*BATCH_SIZE]
+        time.sleep(SLEEP_TIME)
+        builder(gx, names, new_names)
```

Comment:
I think this part is usually fine, since we aren't actually using the gh API. The problem usually comes from the updating of the prs here: https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/make_graph.py#L181